docs/release-notes.rst

```diff
@@ -6,6 +6,196 @@
+
+-  WebUI: Fix a bug where the Compare view on the Project Details page did not allow comparison of
+   experiments selected from different pages.
+-  WebUI: Fix endless metrics fetching in "Visualization" tab in experiment details page for
+   cancelled experiments that do not have metrics.
+-  Fix two places where aggregated queued stats could have shown inflated values. The total queued
+   aggregated time and today's queued aggregated time calculations were both affected.
+-  CLI: Fix an error related to ``det cmd list --csv``
+-  WebUI: Fix missing data in Historic Usage Charts due to erroneous date parsing.
+
+**API:**
+
+-  API: Add new endpoints for creating and managing workspace namespace bindings.
+-  Add POST and DELETE endpoints to ``/api/v1/workspaces/{workspace_id}/namespace-bindings`` for
+   setting and deleting workspace namespace bindings.
+-  Add a GET endpoint ``/api/v1/workspaces/{id}/list-namespace-bindings`` to list namespace bindings
+   for a workspace.
+-  Add a POST endpoint ``/api/v1/workspaces/{id}/set-resource-quota`` to set resource quotas on
+   workspaces bound to auto-created namespaces.
+-  Add a GET endpoint ``/api/v1/workspaces/{id}/get-k8-resource-quotas`` to retrieve enforced
+   Kubernetes resource quotas for bound namespaces
+
+**Deprecations**
+
+-  Detached mode: The ``defaults`` and ``unmanaged`` parameters of the ``init`` function for
+   unmanaged experiment have been deprecated and will be removed in a future version. Please use
+   ``config`` instead.
+
+-  Agent and Kubernetes Resource Manager: Jobs can no longer be moved within the same priority
+   group. To reposition a job, update its priority using the CLI or WebUI. For detailed
+   instructions, visit :ref:`modify-job-queue-cli`. This change was announced in version 0.33.0.
+
+-  AgentRM: Support for Singularity, Podman, and Apptainer was deprecated in 0.33.0 and is now
+   removed. Docker is the only container runtime supported by Agent resource manager (AgentRM). It
+   is still possible to use podman with AgentRM by using the podman emulation layer. For detailed
+   instructions, visit: `Emulating Docker CLI with Podman
+   <https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman>`. You
+   might need to also configure checkpoint_storage in experiment or master configurations: `Master
+   Config Reference
+   https://docs.determined.ai/latest/reference/deploy/master-config-reference.html#checkpoint-storage`
+
+In the enterprise edition, Slurm resource manager still supports Singularity, Podman, or Apptainer
+use. For detailed instructions, visit :ref:deploy-on-slurm-pbs.
+
+Kubernetes Scheduling: Support for the priority scheduler for Kubernetes Resource Managers is
+discontinued and may be removed in a future release due to limited usage. Users should transition to
+the default scheduler. Visit :ref:`Kubernetes Default Scheduler <kubernetes-default-scheduler>` for
+details.
+
+**Deprecation**
```

Comment:
duplicate section title

Reply:
Will remove! Thanks for catching this! I'll make sure any other duplicates are also handled

Reply:
Cleaned up 😄